README.md

```diff
@@ -118,57 +118,57 @@ System requirements can vary depending on the use case. The following are the mi
 
 Here is a general example of running a Cloud Agent locally:
 ```bash
-PORT=${PORT} PRISM_AGENT_VERSION=${PRISM_AGENT_VERSION} PRISM_NODE_VERSION=${PRISM_NODE_VERSION} \
+PORT=${PORT} AGENT_VERSION=${AGENT_VERSION} NODE_VERSION=${NODE_VERSION} \
```

Comment:
hm...
I guess we should consider another suffix for the environment variables and belong to the agent:
IDENTUS_CLOUD_AGENT_VERSION - is too long
ICA_VERSION - is good from my point of view, ICA abbreviation should be introduced.

Reply:
@yshyn-iohk  we are  following the spreadsheet if you have comments  lets discuss here https://docs.google.com/spreadsheets/d/1lR3abdg6qExXasUVqBlFouNu9FCB0I1ynMMZodQ6y8U/edit#gid=0

Reply:
@mineme0110, let's discuss the renaming there